src/paths/openapi/load.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Load an openapi file though IESI to provide components and connections
+tags:
+  - openapi
+operationId: openapiLoad
+security:
+  - bearerAuth: ["OPENAPI_READ, OPENAPI_WRITE"]
```

Comment:
Do we need security on this endpoint?

Reply:
It requires read privilege because at the end the API sends the components and connections that were generated
And if the doucmentation has to be loaded it requires write privileges